xclim/indices/_threshold.py

```diff
@@ -292,8 +292,9 @@ def cold_spell_max_length(
         rl.longest_run,
         freq=freq,
     )
-    out = max_l.where(max_l >= window, 0)
-    return to_agg_units(out, tas, "count")
+    max_window = max_l.where(max_l >= window, 0)
+    out: xarray.DataArray = to_agg_units(max_window, tas, "count")
```

Comment:
Same as for `rate2amount` : isn't `to_agg_units` already annotated correctly?

Reply:
This is true. Reverting.